src/main/resources/public/js/view.js

```diff
@@ -162,23 +225,33 @@ function updateInfo(xint, yint){
 	- x:[0, 12], y:[0, 12]
 	- 
 */
-function setupGridLines() {
+function setupGridLines(scale, x_offset, y_offset) {
     var lines_y = [];
     var lines_x = [];
 
-    for(var i=0; i<25; i=i+1){
+    for(var i=0; i<80; i=i+1){
         lines_y[i] = new PIXI.Graphics();
         lines_y[i].lineStyle(1, 0x0000FF, 1);
-        lines_y[i].moveTo(0,i*20);
-        lines_y[i].lineTo(520,i*20);
-        lines_y[i].x = 0; lines_y[i].y = i*20;
+
+        lines_y[i].moveTo(0,i*65*scale/100);
+        lines_y[i].lineTo(520,i*65*scale/100);
```

Comment:
it would be ncie if you define some constants so it's easier to follow the math throughout the code. Like 
const SCREEN_WIDTH = 520;
const IMAGE_WIDTH = 1300; 

etc.